packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor

```diff
@@ -358,5 +358,21 @@
                 </NimbleTreeItem>
             </NimbleTreeView>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Wafer Map</div>
+            <NimbleWaferMap 
+                OriginLocation="WaferMapOriginLocation.BottomLeft"
+                Orientation="WaferMapOrientation.Bottom"
+                MaxCharacters=2
+                DieLabelsHidden="false"
+                ColorScaleMode="WaferMapColorScaleMode.Linear"
+                @bind-Dies="Dies"
+                @bind-ColorScale="ColorScale"
+                @bind-HighlightedTags="HighlightedTags"
+            >
+            </NimbleWaferMap>
+            <NimbleButton @onclick=@(_ => AddDiesToRadius(10))>Increase Radius</NimbleButton>
+            <NimbleButton @onclick=@(_ => RemoveDiesFromRadius(10))>Decrease Radius</NimbleButton>
```

Comment:
I don't think the labels are super clear in the example. Could we use the labels "Increase Die Count" and "Decrease Die Count"?